python/alibiexplainer/alibiexplainer/anchor_text.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from typing import Callable, List, Dict
+
+import alibi
+import kfserving
+import numpy as np
+import spacy
+from alibiexplainer.explainer_wrapper import ExplainerWrapper
+from alibi.utils.download import spacy_model
+
+logging.basicConfig(level=kfserving.server.KFSERVER_LOGLEVEL)
+
+
+class AnchorText(ExplainerWrapper):
+
+    def __init__(self, predict_fn: Callable, explainer: alibi.explainers.AnchorText,
+                 spacy_language_model: str = 'en_core_web_md', **kwargs):
+        self.predict_fn = predict_fn
+        if explainer is None:
+            logging.info("Loading Spacy Language model for %s", spacy_language_model)
+            spacy_model(model=spacy_language_model)
```

Comment:
Not at present I think. Its an internal download in Spacy for a corpus. Maybe we can do an update in later PR to add a corpus_download and ask Spacy to use local file so it can be.